src/applications/travel-pay/containers/TravelPayStatusApp.jsx

```diff
@@ -65,6 +67,25 @@ export default function App({ children }) {
     [dispatch, userLoggedIn],
   );
 
+  const CLAIMS_PER_PAGE = 10;
+  let displayedClaims = travelClaims;
```

Comment:
If `displayedClaims` isn't defined as state, is it true that the list of displayed claims won't rerender?

I was testing this out locally and ran into an issue where clicking the 'page 2' button resulted in the list being appended to instead of replaced.

Can you check this out locally as well? I think a test to cover this case would be good to have also.

Reply:
> If displayedClaims isn't defined as state, is it true that the list of displayed claims won't rerender?

`travelClaims` are part of the Redux store, so any time that value gets updated, it will cause the component to re-render. Note that `displayedClaims` only remains unchanged when the total claim are under 10. If there's more than 10, `displayedClaims` becomes whatever is paginated.

> instead of seeing only another set of 10 claims, I saw a larger set of 20 claims

my first thought was this could have been an issue with `travelClaims.slice` getting called with pagination values being misaligned, but I'm having trouble reproducing this one. Could you double check this one? If it's still an issue, I'd like to pair so I can see steps to reproduce it

Reply:
I only saw this issue happen after switching branches, but I haven't been able to replicate it since.

Reply:
@hinzed1127, I no longer saw this issue when using the mock data file you provided in Slack... I want to record that although you and I both saw that I experienced this issue locally, it's either:
- intermittent
- a problem with the test data I was using

Reply:
adding to this comment for posterity: when using the [mock data file added here](https://github.com/department-of-veterans-affairs/vets-website/pull/30269/files#diff-c130212374516b78d56a807900240aa1d738c6ef1901cab3bf371d254232d8f0), supplied via `vets-api` and `vets-api-mockdata`, this issue was no longer observed. The leading hypothesis was an issue related to duplicate/bad mock data, but we will have more tests covered around this in E2E/cypress tests